### DISCUSSION TITLE
Is it possible to run Hawtio with connected remote service

### DISCUSSION THREAD

**Discussion by eshepelyuk** [hawtio/hawtio#3730] [Q&A]
Currently connection is only configured via UI, is it possible to run Hawtio with pre configured remote server connected ?

**Comment by tadayosi** [comment]
Hi,

Not sure if it's exactly what you want, but once you've configured a connection with a name (say `serverA`) via the Connect tab UI, you can automatically connect to the server by attaching `?con=serverA` to the Hawtio console URL, like http://localhost:8080/hawtio/?con=serverA.

**Reply by tadayosi** [reply]
That said, I think it can be a good feature enhancement for the Hawtio CLI, so I went ahead and filed an issue here: hawtio/hawtio#3731

**Reply by eshepelyuk** [reply]
> That said, I think it can be a good feature enhancement for the Hawtio CLI, so I went ahead and filed an issue here: hawtio/hawtio#3731

yes the issue describes desired functionality perfectly
